### ISSUE TITLE
i can train nothing follow the instruction

### ISSUE THREAD

**Issue by hrz2000** [ashawkey/stable-dreamfusion#292]
`python main.py --text "a hamburger" --workspace demo/seed0 -O --seed 0`, After execution, nothing like a hamburger was trained, 
If lambda_entropy is not specified, the image is only something in the middle and not in the shape of a hamburger, When the training continues, it will disappear
<img width="1107" alt="image" src="https://github.com/ashawkey/stable-dreamfusion/assets/79900945/189b7200-1b55-484b-8dc5-8e9607e13a55">
<img width="1168" alt="image" src="https://github.com/ashawkey/stable-dreamfusion/assets/79900945/4b3ec427-d00f-4287-b724-2a8f7713b5b6">
following the nootbook,
<img width="1000" alt="image" src="https://github.com/ashawkey/stable-dreamfusion/assets/79900945/8ea07727-bc71-4b6b-8128-f6c4ffb36b03">
`python main.py --text "a hamburger" --workspace demo/seed_0_half_entr -O --seed 0 --lambda_entropy 0.00005`, After execution, nothing like a hamburger was trained also,
it look，
<img width="1156" alt="image" src="https://github.com/ashawkey/stable-dreamfusion/assets/79900945/6ecc616e-c05e-4894-9a2c-4333209a878b">
<img width="1182" alt="image" src="https://github.com/ashawkey/stable-dreamfusion/assets/79900945/56e5e659-31b8-4b18-9c66-81e53556d370">
I really hope someone can help me solve this problem. I haven't seen similar discussions on other issues. If anyone also encounters this problem, welcome to discuss it together
I tried some learning rates and different random seed, which should not be the reason.
<img width="876" alt="image" src="https://github.com/ashawkey/stable-dreamfusion/assets/79900945/9cbc69f6-83db-4f3b-9330-656f1ea95846">
The loss of the former is also very strange

**Comment by hrz2000** [comment]
it maybe the problem of my sd's weights, i download it from huggingface website instead of using cache weight, it may cause some problems
